docs/tutorials/denom-allolist-proposal.md

```diff
@@ -0,0 +1,40 @@
+# Currency Allow list Proposals
+
+Community members, or representatives of projects/protocols are encouraged to submit proposals for allowing currencies to be used in the marketplace.
+
+This document provides instructions for submitting a currency allowlist proposal using the [command-line interface (CLI)](../ledger/infrastructure/interfaces.html#command-line-interface).
+
+## Submitting the proposal via the Command-Line Interface
+
+Before submitting the proposal you will need to format a json using the template below
+
+***proposal.json:***
+```
+{
+    "title": "some title",
+    "description": "some description",
```

Comment:
We should probably provide a better example title and description given this is specific to adding a denom.